README.md

```diff
@@ -299,6 +303,7 @@ Additionally, one can also add Kibana to the mix. In order to do so, one can use
 ### Deploy
 
 ```shell
+kubectl create -f kibana-cm.yaml # create configMaps before deployment
```

Comment:
Please, remove comment. Alternatively, please add documentation on the fact we're using a ConfigMap to configure Kibana, and how can one change it to fit their needs.